community/docker/APKBUILD

```diff
@@ -105,12 +107,33 @@ build() {
 	msg "building tini"
 	cd "$_tini_builddir" || return 1
 	cmake . || return 1
+	sed -i "s/TINI\_GIT *\"[^\"]*/TINI_GIT     \" - git.$_tini_gitver/"  tiniConfig.h
```

Comment:
I think we should report this upstream to tini. As I understand the problem here is that docker will check for a specific git revision, the tini build script will execute `git log -n 1  --pretty=format:%h` to get the git revision of tini, but since we are using a tarball generated from the given git commit, the tini build script fails to set the correct version string.

I think a better solution would be to patch tini to allow override the `tini_VERSION_GIT` from the cmake run. something like:
`cmake -Dtini_VERSION_GIT="$(echo $_tini_gitver | cut -b1-7)"`